src/devices/Card/CreditCard/ConversionType.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Iot.Device.Card.CreditCardProcessing
+{
+    /// <summary>
+    /// The type of conversion for the data conversion.
```

Comment:
what do you mean by `default` here in the description? `default(ConversionType)==BcdToString==0`

Reply:
Corrected, default(ConversionType) ==ByteArray